x-pack/test/apm_api_integration/tests/alerts/helpers/alerting_api_helper.ts

```diff
@@ -38,13 +38,14 @@ export async function createApmRule<T extends ApmRuleType>({
         params,
         consumer: 'apm',
         schedule: {
-          interval: '5s',
+          interval: '1m',
```

Comment:
This change was needed because serverless threw an error if the interval was <1min.
I will monitor and check if all the other stateful tests are still passing

Reply:
Not needed anymore